---
title: "Sanders Lab - Research"
layout: textlay
excerpt: "Sanders Lab -- Research"
sitemap: false
permalink: /research/
---

# Research Highlight

---


![]({{ site.url }}{{ site.baseurl }}/images/respic/asdcnv.png){: style="width: 300px; float: left;margin-right: 30px; border: 10px"}

## Copy number variants (CNVs) in autism spectrum 
<div style="text-align: justify">
My early publications describe the development of a novel approach to detecting de novo CNVs in Illumina SNP genotyping data and applying this to Autism Spectrum Disorder (ASD) families from the Simons Simplex Collection (SSC). While de novo CNVs had previously been associated with ASD (Sebat et al. 2007), our analysis of this cohort of 1,124 families, including unaffected siblings, allowed us to clarify the genomic architecture. In turn, this allowed us to develop a statistical approach to evaluating whether specific genomic loci are associated with ASD; four de novo duplications at 7q11.23 met these criteria for ASD association with reduced sociability and, notably, deletions at the same locus are associated with increased social behavior in Williamâ€™s Syndrome. We have recently (2015) published a CNV analysis of all 2,591 families in the SSC. By integrating CNV and exome data we showed that small de novo deletions contain one critical ASD gene, while large de novo CNVs contain multiple ASD risk genes. We use this finding to identify six ASD risk CNV loci and 65 ASD risk genes. I led the analysis and write up of both projects including the development of novel approaches to CNV detection. Autism Speaks cited our 2011 CNV analysis as one of the top Ten Autism Research Achievements. 
</div>
---

![]({{ site.url }}{{ site.baseurl }}/images/respic/genediscovery.png){: style="width: 300px; float: left;margin-right: 30px; border: 10px"}

## Loss of function (LoF) mutations in autism spectrum disorder 
<div style="text-align: justify">
Our work was the first to show that de novo LoF mutations are observed at a higher rate in ASD cases than unaffected sibling controls, thereby demonstrating association with ASD. In addition, we developed a statistical approach to evaluating the probability of observing multiple de novo LoF mutations in the same gene. This allowed us to identify specific genes associated with ASD, including the sodium channel SCN2A and the chromatin regulator CHD8. I led the analysis and write up of this project including the development of approaches to discovering de novo mutations and the statistical assessment of multiple mutations in the same gene. This research has had a transformative effect on ASD genetics; statistical refinement and the application to larger cohorts has replicated our initial findings and when combined with the CNV data led to the discovery of over 65 ASD genes (Sanders et al. 2015, see previous topic). In addition application of our approach has led to the identification of disease-associated genes in many other disorders, including intellectual disability, epileptic encephalopathy, and congenital heart disease. Both Autism Speaks and SFARI cited our work on de novo LoF mutations as one of the top ten papers of the year.
</div>
---- 

![]({{ site.url }}{{ site.baseurl }}/images/respic/scn2aMut.png){: style="width: 300px; float: left;margin-right: 30px; border: 10px"}

## Understanding the role of ASD genes
<div style="text-align: justify">
Having developed a robust method for identifying ASD genes, we have begun to consider how we can use this foundation to understand the neuropathology of ASD. In collaboration with the [Bender lab at UCSF](https://benderlab.ucsf.edu/research-projects), we demonstrated that loss of function mutations of SCN2A lead to autism and developmental delay, while gain of function mutations lead to epileptic encephalopathy (Ben-Shalom et al. 2017). Similarly, ChIP-Seq analysis of CHD8 shows that this gene acts as the regulator of ASD genes, including SCN2A (Cotney et al. 2015). We have also used co-expression analysis to identify convergence during mid-fetal development in the prefrontal cortex driven by layer 5/6 glutamatergic neurons (Willsey et al. 2015). Along with investigating the molecular neuropathology of ASD we have also considered the relative contributions of the various genetic and environmental risk factors to ASD risk in the population and in cases, highlighting the role of common variation at a population level (Klei et al. 2012).
</div>
